### PR TITLE
Component is working

### LINKED ISSUE #206: crear funcionalidad del perfil de provedor
# EXPLANATION 

Al ingresar con credenciales de proveedor se abre una vista d pantalla donde esta un formulario ,este formulario se debe llenar con los datos para crear un perfil como proveedor pero al ingresar los dato no guarda la información no guarda el imagen  no tiene funcionalidad

# SCREESHOTS

1 Diseño asignado esta es la vista de pantalla de perfil
image

2 Crear la  funcionalidad de logotipo al ingresar un imagen debe guardarse y mostrarse cuando se guarde la infomacion como un logotipo de la empresa en su perfil

image

3 las validaciones  "Cancelar" y "Guardar" deben funcionar

# USER STORY 
Como proveedor quiero ingresar una imagen como logo de la empresa

Como proveedor quiero crea mi perfil al seleccionar la opción "Guardar"

### LINKED ISSUE #264: specifically allow only png image for edit profile
**Explanation**

- When uploading the provider's profile image, it must be in png format, no other format should be supported.

### LINKED ISSUE #265: that the profile picture works properly
**Explanation**

- An enter the profile view the image is not working correctly is not loading

### PR DESCRIPTION
**Pull request by PrototypeK02**
### Description

Please include a summary of the change.

This component is working now. Accepts images, and updates user info

#### Screenshots

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
Please rebase

**Comment by PrototypeK02** [comment]
@Israel-Laguan can you please let me know which error is throwing vercel so I can fix it please?

File: containers/Perfil/ProfileContainers.tsx [hunk 1]
```diff
@@ -1,26 +1,38 @@
...
-    push('/productos')
-  }
+  const [loading, setLoading] = useState<boolean>()
+  const submitHandler = async (values: ProviderEditable, e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    setLoading(true)
 
-  if (mutationError) logError(mutationError, 'pages/perfil/editar/[id].tsx', 'Error al actualizar el perfil')
+    const form = new FormData(e.target as HTMLFormElement)
+    form.append('id', `${user.providerInfo.id}`)
```
Comment (e-azocar) [inline containers/Perfil/ProfileContainers.tsx]:
```suggestion
    form.append('id', user.providerInfo.id)
```

**Review by e-azocar — APPROVED** [review]

File: services/GraphQL/providers/types.d.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export interface ProviderBase {
 
 export interface ProviderEditable extends ProviderBase {
   commercialName?: string
-  logoUrl?: string
+  logo?: string
```
Comment (Israel-Laguan) [inline services/GraphQL/providers/types.d.ts:45]: Why change this type? in the query the name is logoUrl

File: components/UpdateForm/index.tsx [hunk 3]
```diff
@@ -60,7 +61,7 @@ const UpdateFormContainer: FC<Props> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  if (loading) return <Loading className="center-text" />
```
Comment (heliomar-pena) [inline components/UpdateForm/index.tsx]: This class is to center the text while the component is loading, it's causing any problem?

Comment (heliomar-pena) [reply components/UpdateForm/index.tsx]: Currently it looks so:
imagen

But should looks centered

**Review by heliomar-pena — APPROVED** [review]
Good work! Just a min comment

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
Please rebase

**Review by Israel-Laguan — APPROVED** [review]

**Comment by PrototypeK02** [comment]
issues #265 and #264 done in this PR

**Review by e-azocar — APPROVED** [review]

File: services/GraphQL/providers/types.d.ts (hunk 2 above)
Comment (Israel-Laguan) [reply services/GraphQL/providers/types.d.ts:45]: @e-azocar @Eghinner please check

**Comment by angelik0828** [comment]
## The profile picture looks good it changes

image
image


- ⚠️ When the image is uploaded, you can see the text when no image has been added yet.

image

**Comment by Eghinner** [comment]
> * When the image is uploaded, you can see the text when no image has been added yet

@angelik0828 Could you explain 
>⚠️ When the image is uploaded, you can see the text when no image has been added yet.

**Comment by angelik0828** [comment]
## Of course when you upload an image it doesn't take all the image space, and you can see the word upload image even though it's already uploaded, that's what needs to be fixed

**Comment by Eghinner** [comment]
> Of course when you upload an image it doesn't take all the image space, and you can see the word upload image even though it's already uploaded, that's what needs to be fixed

its true. thanks, we could leave it for another issue

**Pull request merged**